packages/web-frontend/src/containers/MeasureBar/Control.js

```diff
@@ -129,6 +130,10 @@ export const Control = ({
     onUpdateMeasurePeriod(moment(startDate).startOf(period), moment(endDate).endOf(period));
   };
 
+  let { startDate, endDate } = selectedMeasure;
+  if (!startDate && defaultDates.startDate) startDate = defaultDates.startDate;
```

Comment:
In the prop types it says that default dates is required. If so then maybe this check is not necessary? However if it is possible for there to be one or the other of startDate and endDate then I can see why the check is needed.

Reply:
Good point, removed